### PR TITLE
[5.0] Joomla Dialog for Aricles, Categories batch and Finder Index with Statistic

### PR DESCRIPTION
**Pull request by Fedik**
### Summary of Changes

An implementation of Joomla Dialog for  for Aricles, Categories batch and Finder Index with Statistic.
Depends from #40150

~It works but not fully finished.~ 
~It still will be need some stilyng for batch modals and solution for 'onclose'~ :potato: https://github.com/joomla/joomla-cms/blob/0659dc263f9461af20ccab361de19796ca9aafbd/administrator/components/com_finder/src/View/Index/HtmlView.php#L183

### Testing Instructions

Apply patch then apply #40150, run `npm install`.
Check Article, Categories batch, and Finder Index wtih Statistic popups. Versions popup

### Actual result BEFORE applying this Pull Request
Works

### Expected result AFTER applying this Pull Request
Works

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: 
- [ ] No documentation changes for docs.joomla.org needed
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/184
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by dgrammatiko** [comment]
>  and solution for `onclose`

That needs to be killed, it's not CSP strict and by default it's an XSS vulnerability as users could execute any arbitrary code using that event. In sort all the inline events should be considered harmful and removed ASAP...

**Comment by HLeithner** [comment]
i think the b/c plugin still has a reference to this es5?

**Comment by Fedik** [comment]
hmhm, what to reference?

**Comment by HLeithner** [comment]
hmm I think I comment on the wrong PR

**Comment by HLeithner** [comment]
thanks, documentation update please

**Comment by Fedik** [comment]
There it is https://github.com/joomla/Manual/pull/184

File: administrator/components/com_finder/src/View/Index/HtmlView.php [hunk 1]
```diff
@@ -183,27 +183,30 @@ protected function addToolbar()
 
             $childBar = $dropdown->getChildToolbar();
 
-            $childBar->popupButton('index', 'COM_FINDER_INDEX')
+            $childBar->popupButton('archive', 'COM_FINDER_INDEX')
```
Comment (Quy) [inline administrator/components/com_finder/src/View/Index/HtmlView.php:186]: Should this be reverted? It is indexing and not archiving.

Comment (Fedik) [reply administrator/components/com_finder/src/View/Index/HtmlView.php:186]: It probably happened while conflict resolving, but it only the button and icon name.
In j4 it archive https://github.com/joomla/joomla-cms/blob/fa61389232ce1870bafeefb892838b4b30ef6b82/administrator/components/com_finder/src/View/Index/HtmlView.php#L177

Comment (Fedik) [reply administrator/components/com_finder/src/View/Index/HtmlView.php:186]: I will make PR, to change it

Comment (Fedik) [reply administrator/components/com_finder/src/View/Index/HtmlView.php:186]: There is PR https://github.com/joomla/joomla-cms/pull/41838

**Pull request merged**